library/std/src/sys/pal/unix/process/process_unix.rs

```diff
@@ -24,7 +24,7 @@ use libc::{c_int, pid_t};
 use libc::{gid_t, uid_t};
 
 cfg_if::cfg_if! {
-    if #[cfg(all(target_os = "nto", target_env = "nto71"))] {
+    if #[cfg(target_os = "nto")] {
```

Comment:
The reason for checking the QNX version here is that this workaround is only required in QNX 7.0 and 7.1. The bug should be fixed with version 8.0. In other words, we should not apply the workaround in QNX 8.0 (which is not yet supported by Rust). Removing the check here will make it a bit more likely that we will miss it later.

Reply:
thx, good to know, fixed